implementations/rust/ockam/ockam_api/src/cloud/secure_clients.rs

```diff
@@ -87,75 +96,116 @@ impl NodeManager {
     }
 
     pub async fn controller_node(
+        context: &Context,
```

Comment:
@davide-baldo I think that you could store the `TcpTransport` inside the client directly since it already contains a copy of the `Context`. Then there's no need to pass a context everywhere to instantiate the clients. It was my original intention that we would only need the `Context` when using the clients, not creating them (at least not explicitly since it is implicitly provided via the Transport...)

Reply:
I could store the transport directly, but I would need to clone it anyway, I can reduce the operation by cloning only the context from the transport, so I can have the best of both world!